libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/internal/DefaultApolloExtension.kt

```diff
@@ -252,6 +254,31 @@ abstract class DefaultApolloExtension(
     }
   }
 
+  @Suppress("ObjectLiteralToLambda")
+  private fun registerCheckLegacyJsTargetTask(): TaskProvider<Task> {
+    return project.tasks.register(ModelNames.checkLegacyJsTarget()) {
```

Comment:
Do we need a task for that? Could we just fail at configuration time, maybe using `afterEvaluate {}` 🙃 ?

Reply:
Haha I guess! I'll have a look.